docs/reference/aggregations/pipeline/serial-diff-aggregation.asciidoc

```diff
@@ -0,0 +1,102 @@
+[[search-aggregations-pipeline-serialdiff-aggregation]]
+=== Serial Differencing Aggregation
+
+experimental[]
+
+Serial differencing (or just differencing) is a technique where values in a time series are subtracted from itself at
```

Comment:
I think we should remove the "(or just differencing)" as it might lead to confusion as to what can be achieved with this agg